prow/cluster/jobs/istio-releases/daily-release/istio-releases.daily-release.collab-gcp-identity.yaml

```diff
@@ -33,73 +33,49 @@ presubmits:
   istio-releases/daily-release:
 
   - name: daily-unit-tests
-    agent: kubernetes
-    context: prow/daily-unit-tests.sh
     branches: *branch_spec
     always_run: true
-    rerun_command: "/test daily-unit-tests"
-    trigger: "(?m)^/(retest|test (all|daily-unit-tests))?,?(\\s+|$)"
     labels:
       preset-service-account: "true"
       preset-istio-kubeconfig: "true"
     spec:
       <<: *build_spec
 
   - name: daily-e2e-pilot-no_auth
-    agent: kubernetes
-    context: prow/daily-e2e-pilot-no_auth.sh
     branches: *branch_spec
     always_run: true
-    rerun_command: "/test daily-e2e-pilot-no_auth"
-    trigger: "(?m)^/(retest|test (all|daily-e2e-pilot-no_auth))?,?(\\s+|$)"
     labels:
       preset-service-account: "true"
     spec:
       <<: *test_spec
 
   - name: daily-e2e-bookinfoTests
-    agent: kubernetes
-    context: prow/daily-e2e-bookinfoTests.sh
     branches: *branch_spec
     always_run: true
-    rerun_command: "/test daily-e2e-bookinfo"
-    trigger: "(?m)^/(retest|test (all|daily-e2e-bookinfo))?,?(\\s+|$)"
     labels:
       preset-service-account: "true"
     spec:
       <<: *test_spec
 
   - name: daily-e2e-simpleTests
-    agent: kubernetes
-    context: prow/daily-e2e-simpleTests.sh
     branches: *branch_spec
     always_run: true
-    rerun_command: "/test daily-e2e-simple"
```

Comment:
This will default to `/test daily-e2e-simpleTests`

Reply:
Yeah that s fine. Prow will tell people what to do on the PR